homeassistant/components/twitter/notify.py

```diff
@@ -63,7 +64,7 @@ def __init__(
         username,
     ):
         """Initialize the service."""
-        self.user = username
+        self.default_user = username
```

Comment:
This can now be reverted to the original

Reply:
But isn't it more accurate to name it default_user? Because ultimately, that's what this is about. If there is no user specified, then the default user will be used instead.